src/entrypoint.sh

```diff
@@ -37,22 +37,11 @@ function parse_inputs {
 }
 
 function install_kustomize {
+    url = "https://github.com/kubernetes-sigs/kustomize/releases/download/kustomize/v${kustomize_version}/kustomize_v${kustomize_version}_darwin_amd64.tar.gz"
```

Comment:
cannot fix to `darwin_amd`.

In case of kustomize, for different versions, the download URL is different and thus had to workaround it like this. Could you provide some test case with v4.x.x, v3.x.x & earlier ?